### PR TITLE
Feat/binf 495 wgs maf

### PR DESCRIPTION
**Pull request by tzuni**
changes to support WGS MAF work

### CODE REVIEW

File: aliquotmaf/merging/record_merger/impl/v1_0.py [hunk 1]
```diff
@@ -40,30 +40,51 @@ def caller_order(self):
...
         in their order of priority.
         """
+        # TODO: add svaba?
         return [
             ("mutect2", "MNP"),
+            ("gatk4_mutect2_pair", "MNP"),
             ("vardict", "MNP"),
             ("pindel", "MNP"),
+            ("sanger_pindel", "MNP"),
+            ("caveman", "MNP"),
```
Comment (kmhernan) [inline aliquotmaf/merging/record_merger/impl/v1_0.py:66]: Did zhenyu etc give you this order? Just want to make sure it was discussed.

File: aliquotmaf/merging/record_merger/impl/v1_0.py [hunk 2]
```diff
@@ -40,30 +40,51 @@ def caller_order(self):
...
-        return ["vardict", "pindel", "mutect2", "muse", "varscan2", "somaticsniper"]
+        return [
+            "vardict",
+            "pindel",
+            "mutect2",
+            "muse",
+            "varscan2",
+            "caveman",
+            "sanger_pindel",
+            "gatk4_mutect2_pair" "somaticsniper",
```
Comment (kmhernan) [inline aliquotmaf/merging/record_merger/impl/v1_0.py:51]: did you forget a `,` here?

**Review by kmhernan — CHANGES_REQUESTED** [review]

File: aliquotmaf/merging/record_merger/impl/v1_0.py (hunk 2 above)
Comment (tzuni) [reply aliquotmaf/merging/record_merger/impl/v1_0.py:51]: yep, looks like

Comment (tzuni) [reply aliquotmaf/merging/record_merger/impl/v1_0.py]: fixed in new commit

**Review by kmhernan — APPROVED** [review]
Approved from my end, but charles may need to see about the failing jenkins.

**Review by kmhernan — APPROVED** [review]

**Review by czyszCTDS — APPROVED** [review]

**Pull request merged**